packs/democratic_csi_nfs/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+app {
+  url    = "https://github.com/democratic-csi/democratic-csi"
+  author = "Tim Gross <tgross@hashicorp.com>"
+}
```

Comment:
@mikenomitch something I noticed here is that we're conflating `app.author` and a hypothetical `pack.maintainer` metadata field. Most package managers have a distinction between the two because they're rarely the same person. Ex. [redis on Debian Buster] is maintained by Chris Lamb, who is not the author of Redis (antirez or Redis Labs).

We also don't include as a requirement any kind of contact info for the pack maintainer. Maybe not as required metadata for all registries but as a requirement for _this_ registry, so that we have someone to ping as the codeowner. (Also maybe not an email address but a GitHub handle here?)

Reply:
Yeah, this was originally the author of the underlying app itself (not the pack). But about 40% of PRs get this "wrong" and do pack author. IDK what value its really providing as author of the underlying app.

Git/GitHub will naturally point us to the maintainer (in a roundabout way), so I'm not sure we really even need it for that purpose.

Maybe its something we should just remove?

Reply:
> But about 40% of PRs get this "wrong" and do pack author.

Most likely because that's what [Writing Packs](https://github.com/hashicorp/nomad-pack/blob/main/docs/writing-packs.md#metadatahcl) recommends. Seems reasonable to me to remove it.

Reply:
Oh wow! Maybe I'm "wrong" then 🙃

Reply:
I've opened https://github.com/hashicorp/nomad-pack/pull/240 to update the docs and/or discuss it more generally.